### ISSUE TITLE
Improve multilingual support

### ISSUE THREAD

**Issue by AaronSadlerUK** [AaronSadlerUK/Our.Umbraco.UmbNav#32]
Currently the multilingual support is clunky to say the least.

It would be nice to be able to choose any enabled language in the node picker, handle the fallbacks gracefully and improve the underlying code for the multilingual part

**Comment by mrrm** [comment]
When I build a menu of item that allow "vary by culture", the menu links are not shown

**Comment by AaronSadlerUK** [comment]
Have you tried passing in the current culture when rendering out?

**Comment by mrrm** [comment]
yes, I've tried. the same issue when reloading cache in the backoffice happens. Maybe it's because I use 2 differents domains(one for the website and other for accessing the backoffice). It's not easy to debug. Do you have an example of a full url? this https://localhost:44373/backoffice/UmbNav/UmbNavEntityApi/GetById?id=1&culture=en-US doesn't work
